www/components/app.jsx

```diff
@@ -105,12 +105,20 @@ class App extends React.Component {
 
   render() {
     let list = this.state.links.map((link, idx) => {
+      let desc = link.body.replace(/^\"/, '').replace(/\"$/, '').replace(/\\\"/g, '"')
```

Comment:
FWIW RSS people should remove redundant quotes from their RSS feeds if they didn't mean to have them. Here is one of the examples: https://thefullsnack.com/rss.xml. :trollface: 

Sometimes quotes are meant to be there. For example: "Không có gì quý hơn độc lập tự do", said President Ho.

Reply:
1) thefullsnack don't want to change his RSS generator :trollface: 
2) Many RSS feeds are broken because people are using Wordpress and wrote their post in HTML. WP pollute their feeds with ton of HTML tags.

From (1) and (2) => We can't ask them to fix their problem. We better fix it our own.